.github/labeler.yml

```
@@ -1,26 +1,4 @@
-  - changed-files:
-    - any-glob-to-any-file: ['refarch-backend/**']
-"Template: EAI":
-  - changed-files:
-      - any-glob-to-any-file: ['refarch-eai/**']
-"Template: Frontend":
-  - changed-files:
-      - any-glob-to-any-file: ['refarch-frontend/**']
-"Template: Webcomponent":
-  - changed-files:
-      - any-glob-to-any-file: ['refarch-webcomponent/**']
+  - head-branch: [ '^bug', '-bug-' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)